src/containers/shared/css/token-table.scss

```diff
@@ -0,0 +1,33 @@
+@import './variables';
+
+@mixin token-table {
+  table {
```

Comment:
Use a class instead of a mixin.  They way you are using this causes the same class to be output twice and using `table` globally.

Reply:
wouldnt declaring as a class also cause it to output twice? since we are still importing the file in two files?